src/js/entities-service/entities/actions.js

```diff
@@ -42,6 +42,17 @@ const _Model = BaseModel.extend({
     SharingUpdated({ attributes }) {
       this.set(attributes);
     },
+    CommentAdded({ author, comment, attributes }) {
+      const commentModel = Radio.request('entities', 'comments:model', {
+        id: comment.id,
+        message: attributes.message,
+        created_at: dayjs.utc().format(),
+        edited_at: null,
+        _clinician: author,
+      });
+
+      this.trigger('comment:added', commentModel);
```

Comment:
Fine as `comment:added? Or should it be more specific like `ws:comment:added`? Wasn't sure.

Reply:
should be `add:comment` present tense `verb:noun`

Reply:
Ah right 👍 , okay switched it to that.